### PR TITLE
[ThreatGrid] - fix bug in threat-grid-get-html-report-by-id command

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-17249)

## Description
Fixed an issue where the *threat-grid-get-html-report-by-id* failed because the file content was sent as a raw-response causing it to fail with mixed and invalid characters.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ShahafBenYakir — APPROVED** [review]
Looks good, waiting for the customer approval

**Comment by GuyAfik** [comment]
> Looks good, waiting for the customer approval

Customer reports it solves the issue, need a force-merge here cause the test for Threat Grid is currently in skipped. [IN140]

**Pull request merged**
